apply-templates.sh

```diff
@@ -35,16 +35,21 @@ for version; do
 	variants="$(jq -r '.[env.version].variants | map(@sh) | join(" ")' versions.json)"
 	eval "variants=( $variants )"
 
+	# TODO somehow make sure this deletes any content we're not generating (without accidentally deleting potentialy generated tarballs for the things we *do* care about)
+
 	for variant in "${variants[@]}"; do
 		export variant
 
 		echo "processing $version/$variant ..."
 
+		mkdir -p "$version/$variant"
+
 		{
 			generated_warning
 			gawk -f "$jqt" Dockerfile-builder.template
 		} > "$version/$variant/Dockerfile.builder"
 
 		cp Dockerfile.template "$version/$variant/Dockerfile"
+		ln -svfT amd64/rootfs.tar.gz "$version/$variant/busybox.tar.gz"
```

Comment:
If each `dist` branch has to make this symlink for their respective architecture anyway, we should probably just leave it off of the main branch.

Reply:
Yeah, that's fair -- done!